ceph/ceph_admin/helper.py

```diff
@@ -574,6 +575,24 @@ def file_or_path_exists(node, file_or_path):
     return True
 
 
+def monitoring_file_existence(node, file_or_path, file_exist=True, timeout=120):
```

Comment:
Thanks for adding this.

Reply:
- please `file_exists` and `timeout` parameters in Doc Args.
- please add explanation to `file_exists` param.
- As per the documentation syntax code and style,
```
    """Method to monitor a file existence.

    Args:
        node: node object where file should be exists
        file_or_path: ceph file or directory path
    Returns:
        boolean
    """
```
```
def func(arg1, arg2):
    """Summary line.

    Extended description of function.

    Args:
        arg1 (int): Description of arg1
        arg2 (str): Description of arg2

    Returns:
        bool: Description of return value

    """
    return True
```
Ref: https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html#example-google